### PR TITLE
am: Add --clear option to collectstatic commands

### PR DESCRIPTION
**Pull request by mamedin**
This option has been added to fix an issue upgrading from AM v1.7.2 to
AM v1.8.0. Because this option is safe and is useful in order to remove
stale static files, it has been added to the Archivematica Dashboard and
Archivematica Storage Service packages.

Connects to https://github.com/archivematica/Issues/issues/360

### CODE REVIEW

File: debs/bionic/archivematica-storage-service/Makefile [hunk 1]
```diff
@@ -8,7 +8,7 @@ PACKBUILD_EXTRA_ARGS ?=
 #PACKBUILD_EXTRA_ARGS ?= "-b 1"
 PACKAGE = archivematica-storage-service
 BRANCH     ?= qa/0.x
-VERSION     ?= 0.13.0
+VERSION     ?= 0.13.1
```
Comment (scollazo) [inline debs/bionic/archivematica-storage-service/Makefile]: I think that the SS version should still be v0.13.0 (as that's the tag used to build the package), and we should use the release field for to indicate that this is the second build of the package. The final name should be something like "archivematica-storage-service_0.13.0-2"

File: debs/bionic/archivematica/Makefile [hunk 2]
```diff
@@ -7,7 +7,7 @@ GPG_ID ?= 0F4A4D31
 PACKBUILD_EXTRA_ARGS ?= 
 #PACKBUILD_EXTRA_ARGS ?= "-b 1"
 BRANCH     ?= qa/1.x
-VERSION     ?= 1.8.0
+VERSION     ?= 1.8.1
```
Comment (scollazo) [inline debs/bionic/archivematica/Makefile]: If this is being changed before the 1.8.1 release, I think it should go on it's own PR

File: debs/xenial/archivematica-storage-service/Makefile [hunk 3]
```diff
@@ -7,9 +7,9 @@ GPG_ID ?= 0F4A4D31
 PACKBUILD_EXTRA_ARGS ?= 
 #PACKBUILD_EXTRA_ARGS ?= "-b 1"
 PACKAGE = archivematica-storage-service
-BRANCH     ?= stable/0.11.x
-VERSION     ?= 0.11.0
-RELEASE       ?= +rc2~xenial
+BRANCH     ?= qa/0.x
+VERSION     ?= 0.13.1
```
Comment (scollazo) [inline debs/xenial/archivematica-storage-service/Makefile]: I think this should be
<pre>
VERSION?=0.13.0
RELEASE=-2
</pre>

File: debs/xenial/archivematica/Makefile [hunk 4]
```diff
@@ -6,9 +6,9 @@ DOCKER_IMAGE  = "debbuild-$(NAME)-$(VERSION)"
 GPG_ID ?= 0F4A4D31
 PACKBUILD_EXTRA_ARGS ?= 
 #PACKBUILD_EXTRA_ARGS ?= "-b 1"
-BRANCH     ?= stable/1.7.x
-VERSION     ?= 1.7.0
-RELEASE       ?= +rc1~xenial
+BRANCH     ?= qa/1.x
```
Comment (scollazo) [inline debs/xenial/archivematica/Makefile:9]: Same than previous comments

File: rpm/archivematica-storage-service/Makefile [hunk 5]
```diff
@@ -1,7 +1,7 @@
 NAME          = archivematica-storage-service
-BRANCH        ?= stable/0.11.x
-VERSION       ?= 0.11.0
-RELEASE       ?= rc1
+BRANCH        ?= qa/1.x
+VERSION       ?= 0.13.1
```
Comment (scollazo) [inline rpm/archivematica-storage-service/Makefile]: Same than previous comments

File: rpm/archivematica/Makefile [hunk 6]
```diff
@@ -3,9 +3,9 @@ RPM_TOPDIR    = "/rpmbuild"
 DOCKER_VOLUME = "/src"
 DOCKER_IMAGE  ?= $(subst ~,-,"rpmbuild-$(NAME)-$(VERSION)")
 
-BRANCH     ?= stable/1.7.x
-VERSION     ?= 1.7.0
-RELEASE       ?= rc1
+BRANCH     ?= qa/1.x
```
Comment (scollazo) [inline rpm/archivematica/Makefile:6]: Same than previous comments

**Review by scollazo — CHANGES_REQUESTED** [review]

**Comment by mamedin** [comment]
Thanks Santi, I changed the versions and revision vars. This way `archivematica*-1.8.0-2-*` and `archivematica-storage-service-0.13.0-2*` packages will be built.

Ready for CR again.

**Review by scollazo — APPROVED** [review]
lgtm!
